README.md

```diff
@@ -127,10 +131,31 @@ Please install the ```tree``` command before moving forward:
         ```
 
    - MacOSX
-        ```
-        install tree - http://apple.stackexchange.com/questions/128876/whats-the-equivalent-of-the-unix-tree-command-on-osx
-        install wget - http://stackoverflow.com/questions/4572153/os-x-equivalent-of-linuxs-wget
-        ```
+       
+        - Option 1:
+        
+          - install tree
+          
+          http://apple.stackexchange.com/questions/128876/whats-the-equivalent-of-the-unix-tree-command-on-osx
+        
+          - install wget 
+          
+          http://stackoverflow.com/questions/4572153/os-x-equivalent-of-linuxs-wget
+       
+        - Option 2 (Via brew ):
```

Comment:
We use the latest MacOS Sierra 10.12.6, but we are not sure in which Mac version the brew command started to be necessary for the process. That's the reason we decided to show the two option the first one without brew and the second one with it.